File: PRAK 404/PRAK404_2310817110011_GHANI-MUDZAKIR.py
keluar = False

while keluar == False:
    print("\nPilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n")
    pilih_program = int(input("Masukan Pilihan : ", ))


    if pilih_program == 1:
        print("Masukan nilai pertama : ", end=" ")
        nilai1 = int(input())
        print("Masukan nilai kedua : ", end=" ")
        nilai2 = int(input())
        print(f"Hasil pertambahan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1+nilai2:.2f}\n")

    elif pilih_program == 2:
        print("Masukan nilai pertama : ", end=" ")
        nilai1 = int(input())
        print("Masukan nilai kedua : ", end=" ")
        nilai2 = int(input())
        print(f"Hasil pengurangan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1-nilai2:.2f}\n")

    elif pilih_program == 3:
        print("Masukan nilai pertama : ", end=" ")
        nilai1 = int(input())
        print("Masukan nilai kedua : ", end=" ")
        nilai2 = int(input())
        print(f"Hasil perkalian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1*nilai2:.2f}\n")

    elif pilih_program == 4:
        print("Masukan nilai pertama : ", end=" ")
        nilai1 = int(input())
        print("Masukan nilai kedua : ", end=" ")
        nilai2 = int(input())
        print(f"Hasil pembagian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {nilai1/nilai2:.2f}\n")

    elif pilih_program == 5:
        print("Terimakasih, telah menggunakan kalkulator Ghani Mudzakir")
        keluar = True

    else:
        print("Input anda salah, silahkan coba lagi")
    

File: PRAK 404/PRAK404_2310817110011_Ghanimudzakir.c
#include <stdio.h>
#include <stdbool.h>

int main()
{
    bool keluar = false;
    int pilih_program;
    float nilai1, nilai2;
    while (keluar== false)
    {
        printf("\nPilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
        printf("Masukan Pilihan : ");
        scanf("%d", &pilih_program);

        if (pilih_program == 1)
        {
            printf("Masukan nilai pertama : ");
            scanf("%f", &nilai1);
            printf("Masukan nilai kedua : ");
            scanf("%f", &nilai2);
            printf("Hasil pertambahan antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, nilai1+nilai2);
        }
        else if (pilih_program == 2)
        {
            printf("Masukan nilai pertama : ");
            scanf("%f", &nilai1);
            printf("Masukan nilai kedua : ");
            scanf("%f", &nilai2);
            printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, nilai1-nilai2);
        }   
        else if (pilih_program == 3)
        {
            printf("Masukan nilai pertama : ");
            scanf("%f", &nilai1);
            printf("Masukan nilai kedua : ");
            scanf("%f", &nilai2);
            printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, nilai1*nilai2);
        }    
        else if (pilih_program == 4)
        {
            printf("Masukan nilai pertama : ");
            scanf("%f", &nilai1);
            printf("Masukan nilai kedua : ");
            scanf("%f", &nilai2);
            printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, nilai1/nilai2);
        }
        else if (pilih_program == 5)
        {
            printf("Terimakasih, telah menggunakan kalkulator Ghani Mudzakir");
            keluar =  true;
        }
        else{
        printf("Input anda salah, silahkan coba lagi.");
        }
    }
    

    return 0;
}



